### PR TITLE
Nerdromere/inserts

### PR DESCRIPTION
**Pull request by nerdromere**
So, created the inserts, added the IF EXISTS, changed the route_id to 25, and everything works on my machine; good luck!

### CODE REVIEW

File: Setup/Insert-Data.sql [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+VALUES(321321, 'Lynnwood Transit', '200th ST SW', '46th Ave W');
+
+INSERT INTO BUS_STOP
+VALUES(654789, 'South Everett', '112th St SE', 'South Everett St');
+
+-- Delete all data from ROUTE before inserting
+-- DELETE FROM ROUTE;
+
+INSERT INTO ROUTE
+VALUES(159357, 'Bothell to Lynnwood', 123132, 321321);
```
Comment (RyanRussell00) [inline Setup/Insert-Data.sql]: The route name should be one word. It's the name that's usually seen on the front of the bus, like "LYNNWOOD" or "BELLEVUE"

**Review by RyanRussell00 — CHANGES_REQUESTED** [review]
Your Create-Database file has become different than the one in master. The one in master has constraints for ROUTE, SCHEDULED, and some other stuff.
Please incorporate your changes with the one from master and request another review.
Thanks!

**Review by RyanRussell00 — CHANGES_REQUESTED** [review]
Please make the foreign keys in SCHEDULED to be constraints and then run your Inserts file again. It should fail because your inserts files are using non-existent route_names in the VISITS inserts.

**Review by RyanRussell00 — APPROVED** [review]
Looks good! Merging into master now. Branch will be closed afterwards. Please don't push to this branch anymore.

**Review by mduw — APPROVED** [review]

**Pull request merged**